eos_data_distribution/ndn/chunks.py

```diff
@@ -76,6 +76,29 @@ def _on_interest(self, o, prefix, interest, face, interestFilterId, filter):
 
         self._send_chunk(data, seg)
 
+    def _registerPrefix(self, prefix, cost=None, controlParameters={},
+                           onInterest=None, onRegisterFailed=None,
+                           onRegisterSuccess=None,
+                           *args, **kwargs):
+        if not cost: cost = self.cost
```

Comment:
so it would be this and the register interest stuff ? yep, makes sense, will move them now.